themes/bootstrap3/js/check_save_statuses.js

```diff
@@ -124,12 +139,10 @@ function checkSaveStatuses(_container) {
         id: idval,
         source: $source.val()
       }, $(ajaxItems[i]));
+    } else {
+      VuFind.emit("save-status-done");
```

Comment:
Is this right? This else is inside a loop that's checking multiple things. Does this mean that if one entry in the list is missing id/source, it will cause a false "done" status to be emitted? Or am I misunderstanding?